src/pages/did/[did].tsx

```diff
@@ -50,22 +58,42 @@ export const getStaticProps: GetStaticProps<Props, { did?: string }> = async ({
       redirect: { destination: `/`, permanent: false },
     }
   }
+  const userResponse = await getVESSUserByDid(formatedDid)
```

Comment:
@KoukiMinamoto 
Use the code below to use react-query's fetchQuery when retrieving data from the API within getStaticProps.
This is to prevent double retrieval of data on the server-side and client-side

https://tanstack.com/query/v4/docs/reference/QueryClient/#queryclientfetchquery
```ts
const user = await queryClient.fetchQuery(['vsUser', formatedDid], () => getVESSUserByDid(formatedDid))
```

Reply:
Thanks! i fixed it